<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnBook">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Book</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkPensioner">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pensioner</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkMinor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Minor</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkAdult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Adult</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Group:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lstGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUndo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAuditorium">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;name&gt;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>21</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>374</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>&amp;System</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuSystem"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnBook</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkAdult</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkPensioner</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkMinor</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>cellPressed(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
